scss/forms/_form-select.scss

```diff
@@ -4,7 +4,7 @@
 // https://primer.github.io/.
 
 .form-select {
-  --#{$prefix}form-select-bg-img: #{escape-svg($form-select-indicator)};
+  --#{$prefix}form-select-bg-img: var(--#{$prefix}select-indicator);
```

Comment:
Missing Boosted mod.
Are we sure we don't want to use `escape-svg` just here like Bootstrap? To avoid discrepancies. And maybe backport it to Bootstrap first if we think this is better?